public/css/icinga/forms.less

```diff
@@ -305,12 +305,15 @@ form.icinga-form .form-controls {
     .button();
   }
 
-  input[type="submit"].btn-remove {
-    .button(@body-bg-color, @color-critical, @color-critical-accentuated);
+  input[type="submit"].btn-remove,
+  button[type="submit"].btn-remove {
+    margin-right: auto;
+    .cancel-button();
```

Comment:
I thought I made it clear. Forget `cancel-button`! It's flawed and doesn't match with the `.button` mixin. Leave the styling as it is please.